src/pipeline-server/docker-compose.pipeline-server.yml

```diff
@@ -103,10 +97,8 @@ services:
     image: postman/newman
     container_name: pipeline-init
     command: run /postman/pipeline.json -e "/postman/env.json" --verbose
-    environment:
-      http_proxy: ${http_proxy}
-      https_proxy: ${https_proxy}
-      no_proxy: ${no_proxy}
+    env_file:
+      - .env
```

Comment:
Same as above, it is recommended to take the proxy settings from the host system.